Panto/index.html

```diff
@@ -57,31 +59,42 @@ <h1>Make your interior more minimalistic & modern</h1>
         <div class="why-us container">
           <h2>Why Choosing Us</h2>
 
-          <div class="why-us-item">
-            <h3>Luxury facilities</h3>
-            <p class="why-us-item-text">
-              The advantage of hiring a workspace with us is that givees you
-              comfortable service and all-around facilities.
-            </p>
-            <a href="#" class="more-info">More Info &rarr;</a>
-          </div>
+          <div class="why-us-items">
+            <div class="why-us-item">
+              <h3>Luxury facilities</h3>
+              <p class="why-us-item-text">
+                The advantage of hiring a workspace with us is that gives you
+                comfortable service and all-around facilities.
+              </p>
+              <div class="more-info-box">
+                <a href="#" class="more-info">More Info</a>
+                <span class="more-info-arrow">&rarr;</span>
+              </div>
+            </div>
 
-          <div class="why-us-item">
-            <h3>Affordable Price</h3>
-            <p class="why-us-item-text">
-              You can get a workspace of the highst quality at an affordable
-              price and still enjoy the facilities that are oly here.
-            </p>
-            <a href="#" class="more-info">More Info &rarr;</a>
-          </div>
+            <div class="why-us-item">
+              <h3>Affordable Price</h3>
+              <p class="why-us-item-text">
+                You can get a workspace of the highest quality at an affordable
+                price and still enjoy the facilities that are oly here.
+              </p>
+              <div class="more-info-box">
+                <a href="#" class="more-info">More Info</a>
+                <span class="more-info-arrow">&rarr;</span>
+              </div>
+            </div>
 
-          <div class="why-us-item">
-            <h3>Many Choices</h3>
-            <p class="why-us-item-text">
-              We provide many unique work space choices so that you can choose
-              the workspace to your liking.
-            </p>
-            <a href="#" class="more-info">More Info &rarr;</a>
+            <div class="why-us-item">
+              <h3>Many Choices</h3>
+              <p class="why-us-item-text">
+                We provide many unique work space choices so that you can choose
+                the workspace to your liking.
+              </p>
+              <div class="more-info-box">
+                <a href="#" class="more-info">More Info</a>
+                <span class="more-info-arrow">&rarr;</span>
+              </div>
```

Comment:
a ux tip: when you are trying to navigate via the link, you expect the arrow to be part of the link as well, but here when you click on the arrow, nothing happens.
try moving the arrow inside `a`